x-pack/filebeat/input/awss3/metrics.go

```diff
@@ -210,6 +210,8 @@ func newInputMetrics(id string, optionalParent *monitoring.Registry, maxWorkers
 type monitoredReader struct {
 	reader         io.Reader
 	totalBytesRead *monitoring.Uint
+
+	trackBytes int64
```

Comment:
I think the naming could be more clear. "trackBytes" does not mean much to me. Maybe...

```
	totalBytesReadMetric *monitoring.Uint
	totalBytesRead       int64
```

Reply:
Done, please see - https://github.com/elastic/beats/pull/41755/commits/91c7d87f20f8efa8797803504af816baefd4a0bb 

I have renamed and added doc comments so the tracking is clear :)